blog/Bitcoin-Cash-Node-2020-planos-para-atualização-de-maio-e-adiante.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: layout.html
+---
+
+<% set('title', 'Bitcoin Cash Node 2020: planos para atualização de maio e adiante.') %>
```

Comment:
I am not sure why this Portguese article appears in this Spanish MR.

Looks redundant to me, since already merged to:

https://bitcoincashnode.org/en/newsroom/Bitcoin-Cash-Node-2020-planos-para-atualiza%C3%A7%C3%A3o-de-maio-e-adiante